src/renderer/views/Watch/Watch.sass

```
@@ -48,23 +48,28 @@
 
       .premiereDate
         color: #FFFFFF
-        background-color: rgba(0, 0, 0, 0.7)
-        width: 400px
+        background-color: rgba(0, 0, 0, 0.8)
         height: 60px
-        border-radius: 5%
+        border-radius: 5px
         position: absolute
-        bottom: 5px
+        bottom: 12px
+        left: 12px
+        display: flex
+        align-items: center
+        padding: 0 12px
 
         .premiereIcon
           float: left
           font-size: 25px
-          margin-top: 12px
-          margin-left: 8px
-          padding: 5px
+          margin: 0 12px
 
         .premiereText
-          margin-left: 50px
-          margin-top: 10px
+          min-width: 200px
+          margin: 0 12px
+
+          .premiereTextTimestamp
+              font-size:0.85em
```

Could you please fix the formatting on this line, the rest of the code looks good.
```suggestion
              font-size: 0.85em
```

---

sure thanks